Doc/whatsnew/3.7.rst

```diff
@@ -2401,6 +2401,10 @@ Changes in the Python API
   instead of a :class:`bytes` instance.
   (Contributed by Victor Stinner in :issue:`21071`.)
 
+* :meth:`ArgumentParser.add_subparsers() <argparse.ArgumentParser.add_subparsers>`
+  has a new ``required`` parameter that defaults to ``False``.
```

Comment:
I think this focuses on the mecanism without explaining the use case.
What do you think of:

```
* :mod:`argparse` subparsers can now be made mandatory by passing ``required=True``
  to :meth:`ArgumentParser.add_subparsers() <argparse.ArgumentParser.add_subparsers>`
```